website/docs/assets/dashboards/explorer.json

```diff
@@ -172,7 +172,7 @@
             "uid": "P1809F7CD0C75ACF3"
           },
           "editorMode": "code",
-          "expr": "sum(rate(http_request_duration_seconds_count{handler=\"/v1/query\"}[1m]))",
+          "expr": "sum(rate(http_request_duration_seconds_count{handler=\"/v1/query\"}[2m]))",
```

Comment:
what would be the context or motivation for the `2m` change? why this is better?